ui/ozone/platform/wayland/wayland_keyboard.cc

```diff
@@ -38,6 +38,10 @@ WaylandKeyboard::WaylandKeyboard(wl_keyboard* keyboard,
       &WaylandKeyboard::Modifiers, &WaylandKeyboard::RepeatInfo,
   };
 
+  auto* engine = static_cast<WaylandXkbKeyboardLayoutEngine*>(
```

Comment:
if xkb common is not used, it will crash :)

Reply:
true, maybe build guard it?

Reply:
done